src/olympia/addons/migrations/0054_update_default_locale_es_to_es-es.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.17 on 2025-01-21 11:41
+
+from django.db import migrations
+
+def forwards_func(apps, schema_editor):
+    Addon = apps.get_model('addons', 'Addon')
+    Addon.unfiltered.filter(default_locale='es').update(default_locale='es-es')
+
+
+def backwards_func(apps, schema_editor):
+    Addon = apps.get_model('addons', 'Addon')
+    Addon.unfiltered.filter(default_locale='es-es').update(default_locale='es')
```

Comment:
Is it possible currently for an `Addon` to have `es-es` as the default_locale? Could this run backwards blow that state away?

Reply:
It's only possible if it was forced or is very, very old. As it turns out we have a single add-on in prod that does this, and it's... the `es-ES` language pack for Firefox.

In the unlikely event that we'd run the backwards migration for `addons` and not the one for `translations` in prod, that add-on would have no name/summary and would need to be manually fixed, which isn't the end of the world since it's really only for its `name` and `summary`. I could also have omitted the backwards migration or make it a no-op, I figured it was better to have a real one for local development.

Reply:
I think the backwards compat is good for the migration, just wanted to understand the scope of the impact. I agree it's probably okay. Thanks for clarifying.